RPIPs/RPIP-34.md

```diff
@@ -0,0 +1,365 @@
+
+
+#### VOP Elements
+* Code - A short, unique, human-readable string identifier that can be used to identify instances of this VOP. Should take the rough form of VOP-SUBJECT-ACTION
+* Name - A longer identifier in plain English. 
+* Description - A description of this VOP.
+* Purpose - Why does this VOP definition exist?
+* Responsible Party - What party is responsible for ensuring this VOP takes place when it is triggered?
+* Voting Group - Which group votes on the outcome of this VOP?
+* Vote Type - What platform and format does this vote use?
+* Triggers - See section below
+* Other Requirements - See section below
+
+##### VOP Triggers
+* An event or situation that triggers this VOP to take place.
+* VOPs MAY have multiple triggers. 
+  * Multiple triggers have an OR relationship unless specified otherwise: Any met trigger can result in a new VOP instance.  
+* Triggers SHALL be considered mandatory unless explicitly designated as optional.
+  * 'At Will' is an exception to this, and is always to be considered optional. 
+  * If a mandatory trigger condition is met, a responsible party SHOULD begin an instance of the VOP.
+
+##### VOP Other Requirements
+* A VOP MAY have more detailed requirements or prerequisites than can be easily summarized into the VOP definition format.
+* Any requirements of this nature SHOULD be detailed under a specific heading in the same or another RPIP. 
+  * This heading should then be referenced in the VOP definition under 'Other Requirements'.
+
+---
+
+### VOP Usage - Instances
+> VOP-GMC-MEMBERSELECT-1 is a single GMC membership selection, with a date, candidates, a vote, and results.
+
+* A VOP instance is an instantiation of a VOP definition.  
+* When a VOP instance is triggered, it SHALL be assigned the next free ID number by the responsible party.
+  * ID numbers are unique per VOP definition, start at 1, and increment by 1. 
+  * VOP IDs MUST be appended to the VOP code with a hyphen.
+
+---
+
+### VOP Usage - Records
+> The outcome of the VOP-GMC-MEMBERSELECT-1 instance is recorded in the appropriate record VOP record section.
+
+* A VOP record entry contains the results of a concluded VOP instance.
+* A VOP record section MUST be included in the RPIP in which a VOP is defined.
+  * The VOP record section SHOULD follow the template defined under the VOP Record Section Template heading of this specification.
+  
+* When a VOP instance concludes, its outcome MUST be recorded in the appropriate VOP record section by the RPIP Editors. 
+  * A VOP Record MUST include the date the VOP instance concluded.
+  * A VOP Record MUST include an appropriate URL link indicating the outcome of the VOP.
+  * A VOP Record MUST include a brief, human-readable summary of the outcome.
+
+---
+
+### VOP Usage - Overview
+> A Living informational RPIP that lists the currently defined VOPs and the RPIPs in which they are defined.
+
+* The VOP overview is a summary of defined VOPs within Rocket Pool.  
```

Comment:
Can VOPs be undefined/removed/obsoleted?

Reply:
FML, Patrick flagged this to be yesterday and I forgot. TL;DR, yes. But then ideally you still want the records somewhere, which is somewhat annoying.

Reply:
Now covered removal and modification.